jobQueue/jobWrapper.js

```diff
@@ -126,10 +126,13 @@ function logJobOnComplete(req, job) {
       }
 
       logObject.ipAddress = activityLogUtil.getIPAddrFromReq(req);
-      const tokenDetails = jwtUtil.getTokenDetailsFromRequest(req);
-      logObject.user = tokenDetails.username;
-      logObject.token = tokenDetails.tokenname;
 
+      /**
+       * we already set UserName and TokenName in req headers when verifying
+       * token
+       */
+      logObject.user = req.headers.UserName;
+      logObject.token = req.headers.TokenName;
```

Comment:
this is good. Can we also delete `getTokenDetailsFromRequest` from `utils/jwtUtil.js` ? I do not think `getTokenDetailsFromRequest` is used anywhere else.